js/client-chat.js

```diff
@@ -521,6 +521,14 @@
 				}
 				return false;
 
+			case 'clearpms':
+				$('.pm-window').each(function () {
+					app.rooms[''].closePM($(this).data('userid'));
+					$(this).find('.inner').empty();
+				});
+				this.add("All PM windows cleared.");
```

Comment:
Also, it might be clearer to write: `All PM windows cleared and closed.`

So you know that "cleared" doesn't just mean "closed".